services/ymax-planner/src/cosmos-rest-client.ts

```diff
@@ -212,6 +213,25 @@ export class CosmosRestClient {
     );
   }
 
+  async getAccountSequence(
```

Comment:
I'm trying to phase out this file in favor of services/ymax-planner/src/cosmos-rpc.ts; can we implement this functionality on top of that instead?
```ts
  await rpc
    .request('abci_query', {
      path: '/cosmos.auth.v1beta1.Query/Account',
      data: toHex(QueryAccountRequest.encode({ address }).finish()),
    })
    .then(respContainer =>
      QueryAccountResponse.decode(decodeBase64(respContainer.response.value)),
    );
```
(and we might be able to simplify such consumption even further; I've asked @michaelfig if we already have e.g. lookup tables or other helpers)